tools/buildsys/src/builder.rs

```diff
@@ -387,7 +397,18 @@ impl DockerBuild {
             ),
             target_build_args: TargetBuildArgs::Kit(KitBuildArgs {
                 kit: kit.to_string(),
+                vendor: manifest.info().kit_vendor().context(error::GraphSnafu)?,
+                local_kits: serde_json::to_string(
+                    &manifest.kit_dependencies().context(error::GraphSnafu)?,
+                )
+                .context(error::MetadataSerializeSnafu)?,
+                external_kit_metadata: read_to_string(
+                    args.external_kits_dir.join("external-kit-metadata.json"),
+                )
+                .context(error::ExternalMetadataReadSnafu)?,
```

Comment:
It's weird to pass this particular blob through environment / buildsys / docker, when kits have lots of other blobs (RPMs) that we just take from the host. IMO `rpm2kit` should just look for and incorporate this file instead of wiring it through like this.